itou/www/apply/urls.py

```diff
@@ -59,12 +65,19 @@
         name="check_prev_applications_for_hire",
         kwargs={"hire_process": True},
     ),
+    # FIXME(alaurent) remove in a week
     path(
         "<uuid:session_uuid>/hire/eligibility",
         submit_views.IAEEligibilityForHireView.as_view(),
         name="eligibility_for_hire",
         kwargs={"hire_process": True},
     ),
+    path(
+        "<uuid:session_uuid>/hire/iae_eligibility",
```

Comment:
Pour être cohérent avec la vue GEIQ :
```suggestion
        "<uuid:session_uuid>/hire/iae-eligibility",
```
Et faudrait se décider une bonne fois pour toutes entre `_` et `-` dans les URL :sweat_smile:.

Reply:
Alors, je crois que c'est bien décidé de mettre des `-`, c'est juste que j'ai une fâcheuse tendance à oublier :sweat: